index.html

```diff
@@ -47,64 +49,63 @@
         <div class="wrapper">
             <div class="mainHeadline">
                 <div class="headlineContent">
-                    <h1>Hey there. I'm Ivan.</h1>
+                    <h1>Hey there. <br> I'm Ivan.</h1>
                     <span>I’m a Software Developer</span>
                     <p>I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
-                    <div class="headlineSocial">
-                        <a href="https://github.com/ivangonzalez224">
-                            <img src="images/gitHub.png" alt="">
-                        </a>
-                        <a href="https://www.linkedin.com/in/iv%C3%A1n-gonzalez-robles-957491275/">
-                            <img src="images/linkedin.png" alt="">
-                        </a>
-                        <a href="https://www.angellist.com/">
-                            <img src="images/angelist.png" alt="">
-                        </a>
-                        <a href="https://twitter.com/ivang2204">
-                            <img src="images/twitter.png" alt="">
-                        </a>
-                        <a href="https://medium.com/">
-                            <img src="images/medium.png" alt="">
-                        </a>
-                        
-                    </div>
-                </div>
-
-                <div class="headlineImage">
-                    <img src="images/illustration.png" alt="">
+                </div>    
+                <div class="headlineSocial">
+                    <a href="https://github.com/ivangonzalez224">
+                        <img src="images/gitHub.png" alt="Git Hub Icon">
+                    </a>
+                    <a href="https://www.linkedin.com/in/iv%C3%A1n-gonzalez-robles-957491275/">
+                        <img src="images/linkedin.png" alt="Linkedin Icon">
+                    </a>
+                    <a href="https://www.angellist.com/">
+                        <img src="images/angelist.png" alt="Angel List Icon">
+                    </a>
+                    <a href="https://twitter.com/ivang2204">
+                        <img src="images/twitter.png" alt="Twitter Icon">
+                    </a>
+                    <a href="https://medium.com/">
+                        <img src="images/medium.png" alt="Medium Icon">
```

Comment:
- [ ] Good job making a little bit of adjustment based on the Figma design 👍 
- As you can see below, you'll agree with me that there are little adjustments let on the extra margin you have from the `nav` top to the background image of the header and where the text is placed. Kindly adjust your design while taking reference from the template provided to have a close replica 👍 

## Your work:
![image](https://github.com/ivangonzalez224/Portfolio-mobile-first/assets/50990925/d39440c2-9bb2-4619-9798-44de7524c5f1)

## Figma:
![image](https://github.com/ivangonzalez224/Portfolio-mobile-first/assets/50990925/658b05a9-72fa-4e1e-b9b2-9a399668ad38)